packages/versions/src/cli.ts

```diff
@@ -0,0 +1,47 @@
+import { bold, red, green } from 'chalk';
+import { execSync } from 'child_process';
+import semver from 'semver';
+
+import { versions } from './index';
+
+export function run(_params: { programName: string }) {
+  const { error, info } = console;
+
+  const errorFooter: string = [
+    '\nYou can install/update them with:',
+    ` ${green('https://github.com/fuellabs/fuelup')}`,
+  ].join('\n');
+
+  let userForc: string | undefined;
+  let userFuelCore: string | undefined;
+
+  try {
+    const reg = /[^0-9.]+/g;
+    userForc = execSync('forc --version').toString().replace(reg, '');
+    userFuelCore = execSync('fuel-core --version').toString().replace(reg, '');
+  } catch (err) {
+    error('Make sure you have Forc and fuel-core installed.');
+    error(errorFooter);
+    throw err;
+  }
+
+  const forcOk = semver.satisfies(userForc, versions.FORC);
+  const coreOk = semver.satisfies(userFuelCore, versions.FUEL_CORE);
```

Comment:
TBH, I'm unsure about this one. It might be too strict.

Should we use something else, like `server.lt()` or `server.minVersion()`?

Reply:
I went with:
 - `semver.gt(userVersion, supportedVersion)` = `exit(0)` w/ caution message
 - `semver.eq(userVersion, supportedVersion)` = `exit(0)` w/ success message
 - otherwise fails = `exit(1)`](https://github.com/FuelLabs/fuels-
 
Here:
https://github.com/FuelLabs/fuels-ts/blob/70c795ee702dd323c6cac2b12d10c9353c972576/packages/versions/src/utils/compareUserVersions.ts#L13-L19

And here:
https://github.com/FuelLabs/fuels-ts/blob/9eb62672f40c3a1795d1624838cfd6a238c6cf8d/packages/versions/src/cli.ts#L40-L57